client/src/components/VerbContainer.jsx

```diff
@@ -1,40 +1,137 @@
 import React, { Component } from 'react';
+import MediaQuery from 'react-responsive';
+import styled from 'styled-components';
 
 import ConjugationForm from './ConjugationForm';
 import TenseHeader from './TenseHeader';
+import TenseHeaderMobile from './TenseHeaderMobile';
+
 
 const DEFAULT_TENSES = {
-  "Presente": true,
-  "Presente - Imperativo": false,
-  "Presente perfecto": false,
-  "Pretérito": true,
+  "Presente": true, 
+  "Presente - Imperativo": false, 
+  "Presente perfecto": false, 
+  "Pretérito": true, 
   "Futuro": true,
-  "Futuro perfecto": false,
-  "Pluscuamperfecto": false,
-  "Imperfecto": false,
-  "Pretérito anterior": false,
+  "Futuro perfecto": false, 
+  "Pluscuamperfecto": false, 
+  "Imperfecto": false, 
+  "Pretérito anterior": false, 
   "Condicional": false,
-  "Condicional perfecto": false,
+  "Condicional perfecto": false, 
   // subjunctive
-  "Presente - Subjuntivo": false,
-  "Presente perfecto - Subjuntivo": false,
+  "Presente - Subjuntivo": false, 
+  "Presente perfecto - Subjuntivo": false, 
   "Futuro - Subjuntivo": false,
   "Futuro perfecto - Subjuntivo": false,
-  "Pluscuamperfecto - Subjuntivo": false,
-  "Imperfecto - Subjuntivo": false,
+  "Pluscuamperfecto - Subjuntivo": false, 
+  "Imperfecto - Subjuntivo": false, 
 }
 
+const StyledContainer = styled.div`
+  width: 100%;
+  height: 100%;
+  overflow: scroll;
+  padding: 0 50px;
+
+  @media (max-width: 900px) {
+  padding: 0 20px;
+  }
+`
+
+const TenseHeaderContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`
+
+const StyledVerbContainer = styled.div`
+  display: flex;
+  align-items: center;
+  margin: 10px 0;
+
+  #infinitive-organizer {
+  display: flex;
+  align-items: center;
+  padding: 5px 0;
```

Comment:
this spacing is terrible. is this required by your style library or can you do normal css indentation?

Reply:
no, I was using a 3rd party VSCode lib to do this and obv it didn't quite nail it. Will fix